server/lib/reporting/base-query.ts

```diff
@@ -10,41 +10,26 @@
  * Find more information about this on the LICENSE file.
  */
 export function Base(pattern: string, filters: any, gte: number, lte: number, allowedAgentsFilter: any = null) {
+  const parsedserverSideQuery = JSON.parse(filters);
+  parsedserverSideQuery?.bool?.must?.push?.({
+    range: {
+      timestamp: {
+        gte: gte,
+        lte: lte,
+        format: 'epoch_millis'
+      }
+    }
+  });
   const base = {
-    // index: pattern,
-
     from: 0,
     size: 500,
     aggs: {},
     sort: [],
     script_fields: {},
-    query: {
-      bool: {
-        must: [
-          {
-            query_string: {
-              query: filters,
-              analyze_wildcard: true,
-              default_field: '*'
-            }
-          },
-          {
-            range: {
-              timestamp: {
-                gte: gte,
-                lte: lte,
-                format: 'epoch_millis'
-              }
-            }
-          }
-        ],
-        must_not: []
-      }
-    }
+    query: parsedserverSideQuery
   };
-
   //Add allowed agents filter
-  if(allowedAgentsFilter?.query?.bool){
+  if (allowedAgentsFilter?.query?.bool) {
     base.query.bool.minimum_should_match = allowedAgentsFilter.query.bool.minimum_should_match;
     base.query.bool.should = allowedAgentsFilter.query.bool.should;
   }
```

Comment:
**thought**: if the parsedServerSideQuery contains the filters related to the allowed agents feature, we could be duplicating the query. I checked logging the query, see screenshot:

![image](https://github.com/wazuh/wazuh-kibana-app/assets/34042064/83a56528-3054-4fe5-9b94-dc4327c04ad6)